### PR TITLE
[ SIRIUS ] Update to the latest version.

### PR DESCRIPTION
**Pull request by mtaillefumier**
- SIRIUS supports UPF v2 files now
- various fixes for U+V corrections

### CODE REVIEW

File: tools/toolchain/scripts/stage8/install_sirius.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [ "${BASH_SOURCE[0]}" ] && SCRIPT_NAME="${BASH_SOURCE[0]}" || SCRIPT_NAME=$0
 SCRIPT_DIR="$(cd "$(dirname "$SCRIPT_NAME")/.." && pwd -P)"
 
-sirius_ver="7.5.2"
+sirius_ver="7.6.0"
 sirius_sha256="9ae01935578532c84f1d0d673dbbcdd490e26be22efa6c4acf7129f9dc1a0c60"
```
Comment (oschuett) [inline tools/toolchain/scripts/stage8/install_sirius.sh]: The new sha256 is: `e424206fecb35bb2082b5c87f0865a9536040e984b88b041e6f7d531f8a65b20`.

**Review by oschuett — COMMENTED** [review]
I've uploaded [pugixml-1.14.tar.gz](https://www.cp2k.org/static/downloads/pugixml-1.14.tar.gz) and [SIRIUS-7.6.0.tar.gz](https://www.cp2k.org/static/downloads/SIRIUS-7.6.0.tar.gz).

File: tools/toolchain/scripts/stage8/install_pugixml.sh [hunk 2]
```diff
@@ -0,0 +1,106 @@
+#!/bin/bash -e
+
+# TODO: Review and if possible fix shellcheck errors.
+# shellcheck disable=all
+
+[ "${BASH_SOURCE[0]}" ] && SCRIPT_NAME="${BASH_SOURCE[0]}" || SCRIPT_NAME=$0
+SCRIPT_DIR="$(cd "$(dirname "$SCRIPT_NAME")/.." && pwd -P)"
+
+pugixml_ver="1.14.0"
```
Comment (oschuett) [inline tools/toolchain/scripts/stage8/install_pugixml.sh]: Please change to `1.14` to match pugixml's official release version: https://github.com/zeux/pugixml/releases/tag/v1.14

File: tools/toolchain/scripts/stage8/install_sirius.sh (hunk 1 above)
Comment (mtaillefumier) [reply tools/toolchain/scripts/stage8/install_sirius.sh]: thanks

**Comment by jpoto** [comment]
I don't know where the problem exactly is, but replacing the 'install  ..' commands by 'make install 'and the following changes:
'''
export SIRIUS_CFLAGS="IF_CUDA(-I${pkg_install_dir}/include/cuda|-I${pkg_install_dir}/include -I${pkg_install_dir}/include/sirius)"
export SIRIUS_FFLAGS="IF_CUDA(-I${pkg_install_dir}/include/cuda|-I${pkg_install_dir}/include -I${pkg_install_dir}/include/sirius)"
'''
seem to fix the problem.

**Comment by mkrack** [comment]
@mtaillefumier The new SIRIUS bug fix release 7.6.1 is uploaded to the [CP2K download area](https://www.cp2k.org/static/downloads/). Some prettify issues remain.

**Comment by mkrack** [comment]
Merged in spite of a single slow test input after each `pdbg` test run, because different test inputs were detected as slow during these runs which is rather an indication of noise.

**Pull request merged**